subtitles.md

```diff
@@ -677,3 +677,36 @@ in segment 7.2 "Syntax and semantics of the subtitling segment" of ETSI EN 300 7
 Each Matroska Block **SHOULD** have a Duration indicating how long the DVB Subtitle Segments
 in that Block **SHOULD** be displayed.
 
+## ARIB (ISDB) subtitles
+
+The specifications for the ARIB B-24 subtitle bitstream format (short: ARIB subtitles)
+and its storage in MPEG transport streams can be found in the documents
+"ARIB STD-B24 - Data Coding and Transmission Specification for Digital Broadcasting",
+"ARIB STD-B10 - Service Information for Digital Broadcasting System",
+and "ARIB TR-B14 - Operational Guidelines for Digital Terrestrial Television Broadcasting".
```

Comment:
You should have normative links to these specifications ? They should be added to `rfc_backmatter_codec.md`.

The link being probably https://www.arib.or.jp/english/std_tr/broadcasting/desc/std-b24.html and such.